package.json

```diff
@@ -141,6 +141,7 @@
         "setup": "yarn install",
         "start": "yarn setup && yarn build:dev --watch",
         "start:dev": "yarn build:i18n && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev yarn webpack-dev-server --config build/webpack.config.js",
+        "start:dev:linked": "f() { yarn link ${@:-box-annotations} && IS_LINKED=1 yarn start:dev; }; f",
```

Comment:
Should this be `yarn start:linked`?

Reply:
Maybe I can add another for `yarn start:linked`